taxi/views.py

```diff
@@ -87,3 +106,25 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverForm
+    success_url = reverse_lazy("taxi:driver-list")
+    template_name = "taxi/driver_form.html"
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
```

Comment:
Code Efficiency: The form used in the `DriverUpdateView` is `DriverLicenseUpdateForm`. According to the checklist, you should validate `license_number` while creating and updating users. Please ensure that this form is doing this validation.